src/routes/RemediationDetails.js

```diff
@@ -81,6 +83,11 @@ class RemediationDetails extends Component {
         if (isBeta) {
             this.loadRemediationStatus();
         }
+
+        window.insights.chrome.auth.getUser().then(({ entitlements }) => this.setState({
```

Comment:
Since the function is marked as async you can use `await` in here.
```JS
const { entitlements } = await window.insights.chrome.auth.getUser();
this.setState({ isEntitled: entitlements.smart_management.is_entitled });
```